lm_eval/tasks/xquad.py

```diff
@@ -0,0 +1,299 @@
+"""
+On the cross-lingual transferability of monolingual representations
+https://arxiv.org/abs/1910.11856
+
+XQuAD (Cross-lingual Question Answering Dataset) is a benchmark dataset 
+for evaluating cross-lingual question answering performance. The dataset 
+consists of a subset of 240 paragraphs and 1190 question-answer pairs from 
+the development set of SQuAD v1.1 (Rajpurkar et al., 2016) together with 
+their professional translations into ten languages: Spanish, German, Greek, 
+Russian, Turkish, Arabic, Vietnamese, Thai, Chinese, and Hindi. 
+Consequently, the dataset is entirely parallel across 11 languages.
+
+Homepage: https://github.com/deepmind/xquad
+"""
+import datasets
+from math import exp
+from lm_eval.base import rf, Task
+from functools import partial
+from packaging import version
+from lm_eval.base import PromptSourceTask
+
+
+_CITATION = """
+@article{Artetxe:etal:2019,
+      author    = {Mikel Artetxe and Sebastian Ruder and Dani Yogatama},
+      title     = {On the cross-lingual transferability of monolingual representations},
+      journal   = {CoRR},
+      volume    = {abs/1910.11856},
+      year      = {2019},
+      archivePrefix = {arXiv},
+      eprint    = {1910.11856}
+}
+"""
+
+
+def _squad_metric(predictions, references):
+    squad_metric = datasets.load_metric("squad_v2")
+    return squad_metric.compute(predictions=predictions, references=references)
+
+
+def _squad_agg(key, items):
+    predictions, references = zip(*items)
+    return _squad_metric(predictions=predictions, references=references)[key]
+
+
+class xquad_en(PromptSourceTask):

```

Comment:
We can probably camel-case this to roughly match the task class naming scheme in other files.